.env

```diff
@@ -1,2 +1,2 @@
 VITE_SOLID_IDENTITY_PROVIDER_PRODUCTION="https://solidcommunity.net/"
-VITE_SOLID_IDENTITY_PROVIDER_DEV="https://solidcommunity.net/"
+VITE_SOLID_IDENTITY_PROVIDER_DEV="http://localhost:3000/"

```

Comment:
Really minor, but have noticed local pod URL was misconfigured after Adam mentioned it on Discord.

Even though you could still log into PASS just fine with `http://localhost:3000` and it will correct the right webId, it will generate a webId without the `/` at the end of the Solid Server (notice localhost:3000profile without the slash). Even though the final webId would still need to be altered to http://localhost:3000/test/profile/card#me, could easily fix the localhost:3000profile problem by including `/` to VITE_SOLID_IDENTITY_PROVIDER_DEV.

![Screen Shot 2023-06-08 at 22 31 56](https://github.com/codeforpdx/PASS/assets/14917816/2b9bd2f1-9412-46ec-af64-4ebfd175a9ca)